lightning_pose_app/ui/project.py

```diff
@@ -541,6 +541,8 @@ def get_keypoints_from_zipfile_(file_path, project_type):
 
 def _render_streamlit_fn(state: AppState):
 
+    st_keypoints = []
+
```

Comment:
this fixes the immediate issue, but is more of a bandaid than a fix to the underlying problem. the issue is that when you click "Create new project from source" the flag `state.st_upload_existing_project` is set to True, and when you click another option (say, "Create new project") that flag does not get reset. So instead of declaring `st_keypoints` to be an empty list here the better approach is to set `state.st_upload_existing_project` to False whenever any of the other options are selected.
* after `if st_mode == LOAD_STR` on line 659
* on line 715, `if st_mode == UPLOAD_STR`, add an "else" statement that just contains a block where the flag is set to False
* after `if st_mode == DELETE_STR` on lin 717